Wikipedia/View Controllers/WebView/WebViewController.m

```diff
@@ -378,6 +414,10 @@ - (CGFloat)currentVerticalOffset {
     return self.webView.scrollView.contentOffset.y;
 }
 
+- (BOOL)isWebContentVisible {
+    return CGRectIntersectsRect(self.webView.scrollView.wmf_contentFrame, self.webView.wmf_browserView.frame);
```

Comment:
add `contentInset.top`? fix bug where we try to scroll when webview is barely on screen